src/tools/tidy/src/issues.txt

```diff
@@ -929,6 +920,15 @@
 "ui/drop/issue-90752-raw-ptr-shenanigans.rs",
 "ui/drop/issue-90752.rs",
 "ui/drop/issue-979.rs",
+"ui/dropck/issue-24805-dropck-itemless.rs",
```

Comment:
Why this PR change this file, seems not relevant with it.

Reply:
It's tidy being dumb

Reply:
@workingjubilee: Are you aware of any reason why this would bless differently? All I ran was tidy.

Reply:
seems only the file order is changed, so the root cause may be different behavior on different machines.

it should be sorted :)

Reply:
No idea, I used BTreeSet specifically so this wouldn't happen! :sob:

...Maybe I'm messing up by just iterating from the Difference iterator? I guess the blessed strings need to be collected to a Vec and then sorted to prevent this.

Reply:
yes

```rust
   for filename in allowed_issue_names.difference(&remaining_issue_names) {
            write!(blessed_issues_txt, "\"{filename}\",\n").unwrap();
        }
```

Reply:
Huh? `difference()` should be sorted.

Reply:
yes, it's sorted from `difference(), the interesting thing is I can not reproduce it locally after I added similar test case files.

Reply:
I reproduced it succesfully, will give a fix.

Reply:
There is no issue in the code, but the current issues.txt is not in the right order because of https://github.com/rust-lang/rust/pull/121248#issuecomment-1975274263

Reply:
🤦 we should enforce the list is sorted as part of tidy

Reply:
Good idea, I will add it!